src/app/Layouts/defaultPageWrapper.jsx

```diff
@@ -0,0 +1,44 @@
+import React, { Fragment } from 'react';
+import { node, string, shape } from 'prop-types';
+import Helmet from 'react-helmet';
+import HeaderContainer from '../containers/Header';
+import FooterContainer from '../containers/Footer';
+import { ServiceContextProvider } from '../contexts/ServiceContext';
+import { RequestContextProvider } from '../contexts/RequestContext';
+import GlobalStyle from '../lib/globalStyles';
+import ConsentBanner from '../containers/ConsentBanner';
+
+const PageWrapper = ({ bbcOrigin, children, data }) => {
+  const { isAmp, service } = data;
+  return (
+    <Fragment>
+      <GlobalStyle />
+      <RequestContextProvider
+        platform={isAmp ? 'amp' : 'canonical'}
+        bbcOrigin={bbcOrigin}
+      >
+        <ServiceContextProvider service={service}>
+          <Helmet>
+            <link rel="manifest" href={`/${service}/articles/manifest.json`} />
+          </Helmet>
+          <ConsentBanner />
+          <HeaderContainer />
+          {children}
+          <FooterContainer />
+        </ServiceContextProvider>
+      </RequestContextProvider>
+    </Fragment>
+  );
+};
+
+PageWrapper.propTypes = {
+  bbcOrigin: string,
+  children: node.isRequired,
+  data: shape.isRequired,
+};
+
+PageWrapper.defaultProps = {
+  bbcOrigin: 'https://www.bbc.co.uk',
```

Comment:
I dont think this is right, its left as null https://github.com/bbc/simorgh/blob/latest/src/app/containers/Article/index.jsx#L72 as the logic of the defaulting is handled in https://github.com/bbc/simorgh/blob/latest/src/app/contexts/RequestContext/getOriginContext.js

Adding the default here, the client will always think its on `.co.uk`

Reply:
Oops, yup. fixing.